pywatts/modules/models/profile_neural_network.py

```diff
@@ -76,7 +76,7 @@ def set_params(self, epochs=None, offset=None, batch_size=None, validation_split
         if validation_split:
             self.validation_split = validation_split
 
-    def transform(self, historical_input, calendar, temperature, humidity, profile, trend) -> xr.DataArray:
+    def transform(self, **kwargs) -> xr.DataArray:
         """
         Forecast the electrical load for the given input.
 
```

Comment:
Do you want to leave the list of the inputs (in lines 84 to 95) as it is or do you want to change it kwargs accordingly?

Reply:
I see your point. That's fine.